app/controllers/plantings_controller.rb

```diff
@@ -27,17 +28,6 @@ def index
     end
   end
 
-  # GET /plantings/1
-  # GET /plantings/1.json
-  def show
-    @planting = Planting.includes(:owner, :crop, :garden, :photos).friendly.find(params[:id])
```

Comment:
Drat it. Good catch!

OTOH, I'm surprised this wasn't caught by tests - we should fix that.